webpack.config.js

```diff
@@ -36,6 +37,8 @@ module.exports = {
     new HtmlWebpackPlugin({
       template: './src/index.html'
     })
+    new MomentLocalesPlugin({
+            localesToKeep: ['es-us'],
```

Comment:
![hmm](https://media.giphy.com/media/a5viI92PAF89q/giphy.gif)

Reply:
I don't know how that got there its gone now